build-system/tasks/check-links.js

```diff
@@ -135,7 +135,12 @@ function filterWhitelistedLinks(markdown) {
   filteredMarkdown = filteredMarkdown.replace(/<code>(.*?)<\/code>/g, '');
 
   // Dailymotion link that is marked dead in Travis CI
-  filteredMarkdown = filteredMarkdown.replace(/https:\/\/developer.dailymotion.com\/player#player-parameters/g, '');
+  filteredMarkdown = filteredMarkdown.replace(
+    /https:\/\/developer.dailymotion.com\/player#player-parameters/g, '');
```

Comment:
Nit unrelated to this PR in particular (issue existed before): Any reason why we're not escaping dots? These would match stuff like "https://fast fonts net".